src/main/java/pl/arturzgodka/Diablo3CompediumApplication.java

```diff
@@ -21,7 +22,7 @@ public class Diablo3CompediumApplication {
     public static void main(String[] args) {
         SpringApplication.run(Diablo3CompediumApplication.class, args);
 
-        AccountHandlerApi accountHandlerApi = new AccountHandlerApi();
+        /*AccountHandlerApi accountHandlerApi = new AccountHandlerApi();
```

Comment:
why all the commented out code?

Reply:
I am in the phase of manual testing generated view pages, so to make it faster it's commented out because I often rebuild my project.

Reply:
Now, after restructurizing architecture of project and classes it is faster because not all data are getting from api in the same time.